clang/lib/AST/DeclCXX.cpp

```diff
@@ -1389,7 +1389,11 @@ bool CXXRecordDecl::isLiteral() const {
                              : hasTrivialDestructor()))
     return false;
 
-  if (isLambda() && !LangOpts.CPlusPlus17)
+  // Lambdas are literal types since C++17.
```

Comment:
So why is `isAggregate` or `hasConstexprNonCopyMoveConstructor`/etc not just `return true` here?  Am I missing something?